Project.toml

```diff
@@ -3,20 +3,21 @@ uuid = "cf896787-08d5-524d-9de7-132aaa0cb996"
 version = "5.1.1-DEV"
 
 [deps]
-Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
-SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
+InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 LibGit2 = "76f85450-5226-5b5a-8eaa-529ad045b433"
+Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
+SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 Serialization = "9e88b42a-f829-5b0c-bbe9-9e923198166b"
 Sockets = "6462fe0b-24de-5631-8697-dd941f90decc"
 UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
-Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
-
-[extras]
-Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 
 [compat]
 julia = "1"
 
+[extras]
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+
```

Comment:
I remember we discussed this before, but can't we live with these automatic formattings by Pkg.jl ? I want to add packages via Pkg.jl's interface (which automatically involves these sorting formatting)

Reply:
We should add commands to the extension that make it easier to manage the Project.toml :) Lets see what happens if I next tag things, whether the tagging infrastructure changes the formatting back to something else.